adr/2022_03_18_raw_data_in_command_and_events.md

```diff
@@ -0,0 +1,41 @@
+# Use raw data in commands and events
+
+* Status: accepted
+* Date: 2022-03-18
+
+## Context and Problem Statement
+
+While defining new commands or events we may define them with entities or any objects. Objects passed to commands or
```

Comment:
This is more of my semantical taste, but this section seems to have mixed context and description of the proposed solution. I'm fine with it as it is, but what could we consider for this (or future) ADRs is to focus in this section on the problem itself, like:
```
While working with commands and handlers, we are always working with objects; therefore, these objects need to be present in handlers. We may either fetch them from the repository based on identifiers or pass them as a part of commands. The selected solution needs to take into account that our commands may be dispatched to external systems or possessed asynchronously
```

and then split the current content that is written here into a description of each possible choice